docs/docker/example.md

```diff
@@ -0,0 +1,132 @@
+# Example of creating a development environment with Docker
+
+This project come with a set of small projects that are bundled together by using Docker.
+
+## Dockerfile
+
+Using an image in a raw way can be useful for quick prototyping. However sometimes when you need to do more things internally e.g installing other packages or running command that need to be done before launching your project. a Dockerfile allow you to add more process overhead on top of a based image.
+
+### Example with the API
+
+In this project the API is made with Go a programming language made by Google. For this project we use the ```alpine``` image. Alpine is a lightweight linux environment. In order to run our API. Our API need some third party plugins.
```

Comment:
This project's API is made with Go, a programming language made by Google. For this project, we will use the ```alpine``` image. Alpine is a lightweight linux environment. In order to run our API, we need some third party plugins.

Reply:
Done